### ISSUE TITLE
We should order INPUT from user from the most required (first entries) to the latest advanced less required

### ISSUE THREAD

**Issue by MalloZup** [SUSE/habootstrap-formula#63]
This is more a usability isssue.
I think in order to favor the user experience we should put the most needed to configure variable on the top of list. 
The less and more advanced things should go on the last entry.

![image](https://user-images.githubusercontent.com/10886597/95513241-4f10cc00-09ba-11eb-81ee-9c024d0098c7.png)

**Comment by smpsgill** [comment]
I agree. Same for other SAP forms. I think we can group by logic. So far I followed what was in pillar, and it was more or less order by logic, but we can improve

**Comment by MalloZup** [comment]
@Simranpal  yop. we should  all this work before we release the formula, so we don't give different users experience layout changes

**Comment by jniedergang** [comment]
Hello, I'm sorry I didn't see this request before writing another one.
I have suggested the following layout for a better user experience:
https://github.com/SUSE/habootstrap-formula/issues/79
